src/app/utils/utils.ts

```diff
@@ -182,4 +182,8 @@ export function getPaletteEntryByColorRef(palettes: Palette[], ref: string): Pal
     return  _.minBy(_.filter(paletteList, (paletteEntry => {
         return paletteEntry.enabled && paletteEntry.ref === ref;
     })));
+}
+
+export function colorIsLight(color: Color) {
```

Comment:
Hate the name of this function actually @maxcleme - don't suppose you have a better suggestion? :)

Reply:
I would better do something like : 

```typescript
export function foreground(background: Color): Color
```

Of course it would returns either Black or White, but it makes more sense IMO.

Reply:
Yeah that does make sense actually. It will also move any required if statements out of any dependant code. I'll make this change in a bit. 👍

Reply:
Pushed this change now. Thanks for the tips.